diff --git a/src/seccomp.c b/src/seccomp.c
index ebf3ea1c..02025295 100644
--- a/src/seccomp.c
+++ b/src/seccomp.c
@@ -188,6 +188,10 @@ enable_sandbox_full(void)
 #ifdef TIOCGWINSZ
 	// musl libc may call ioctl TIOCGWINSZ when calling stdout
 	ALLOW_IOCTL_RULE(TIOCGWINSZ);
+#endif
+#ifdef TCGETS
+	// glibc may call ioctl TCGETS when calling stdout
+	ALLOW_IOCTL_RULE(TCGETS);
 #endif
 	ALLOW_RULE(lseek);
  	ALLOW_RULE(_llseek);
